inc/class-sitemaps-index.php

```diff
@@ -17,14 +17,22 @@ public function bootstrap() {
 		add_action( 'init', array( $this, 'url_rewrites' ), 99 );
 		add_filter( 'redirect_canonical', array( $this, 'redirect_canonical' ) );
 		add_action( 'template_redirect', array( $this, 'output_sitemap' ) );
+
+		$core_sitemaps_posts = new Core_Sitemaps_Posts();
+		add_action( 'init', array( $core_sitemaps_posts, 'url_rewrites' ), 99 );
```

Comment:
We need to have discuss an approach where sitemap providers can register themselves without adding any code to other classes, this is where a Core_Sitemap class is needed, and is making things more difficult, so that would be a good next task.

Reply:
If we're not using the index class as the main class, then we should probably move the bootstrap call and the registry instantiation into the main core-sitemaps.php file, later replaced by the Core_Sitemaps class there, and keep it out of Index altogether.

Reply:
Nevermind the previous comment, it doesn't matter for now as it will get replaced with a Core_Sitemaps class.